hkm/models/models.py

```diff
@@ -190,16 +190,20 @@ class Record(OrderedModel, BaseModel):
     edited_preview_image = models.ImageField(verbose_name=_(u'Edited preview image'), null=True, blank=True,
                                              upload_to=get_collection_upload_path)
 
+    """This is populated by view logic if the Record needs
+    its matching Finna data."""
+    finna_entry = None
```

Comment:
Do you mean that I could remove `finna_entry` here and still just use it in `views.py` like it's now done and it will work? Wouldn't that be confusing if the records are being looped in the view and some parts of it will just call `record.finna_entry` which just magically appears from somewhere (from `views.py` in this case)?

Reply:
Yeah, I would remove it, and maybe add a comment to where it is actually used, but I guess this thing is more a matter of opinion and this works too, so forget my initial comment 🙂

Reply:
OK, I'll leave it as it is then 😁